gmail_yaml_filters/ruleset.py

```diff
@@ -598,7 +599,7 @@ def ruleset_to_etree(ruleset):
         'apps': 'http://schemas.google.com/apps/2006',
     })
     etree.SubElement(xml, 'title').text = 'Mail Filters'
-    for rule in sorted(ruleset):
+    for rule in ruleset:
```

Comment:
I'm open to removing the sorting. I am scratching my head trying to remember why I did that in the first place, but I can't recall. Might've had something to do with removing duplicates? 🤷  I'd love help coming up with a test for #14 so we can ensure this doesn't regress at some point in the future.